lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -132,6 +132,36 @@ SdfPath makeDstPath(const SdfPath& dstRootParentPath, const SdfPath& srcPath)
     return dstRootParentPath.AppendPath(relativeSrcPath);
 }
 
+//------------------------------------------------------------------------------
+//
+// Verify if the given prim under the given UFE path is already edited or
+// is an ancestor of an already edited prim.
+bool isAlreadyEditedAsMaya(const Ufe::Path& ufePulledPath)
+{
+    MObject pullSetObj;
+    auto    status = UsdMayaUtil::GetMObjectByName(kPullSetName, pullSetObj);
+    if (status != MStatus::kSuccess)
+        return false;
+
+    MFnSet         fnPullSet(pullSetObj);
+    MSelectionList members;
+    const bool     flatten = true;
+    fnPullSet.getMembers(members, flatten);
+
+    for (unsigned int i = 0; i < members.length(); ++i) {
+        MDagPath itemPath;
```

Comment:
I know this is a frequent problem for us, because of the various paths we have to deal with (Maya, Sdf, UFE), but the variable naming here had me confused.  I would rename the variables like this:
ufePulledPath --> ufeQueryPath (it hasn't been pulled yet)
itemPath --> pulledDagPath
itemUfePath --> pulledUfePath